<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 41, 47);</string>
  </property>
  <widget class="QLineEdit" name="Email_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>128</y>
     <width>350</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(74, 79, 90);
font: 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Qrcode_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>128</y>
     <width>215</width>
     <height>215</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Recover_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>205</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgba(255, 255, 255, 0);
	color: rgb(182, 182, 182);
}
QPushButton:hover
{		
	color: rgb(220, 220, 220);
}</string>
   </property>
   <property name="text">
    <string>Can't sign in? Recover access now</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>195</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 182, 182);</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="Or_by_qrcode_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>104</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(25, 180, 255);</string>
   </property>
   <property name="text">
    <string>OR BY QR-CODE</string>
   </property>
  </widget>
  <widget class="QLabel" name="Remember_me_label">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>262</y>
     <width>111</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(182, 182, 182);</string>
   </property>
   <property name="text">
    <string>Remember me</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>214</y>
     <width>350</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(74, 79, 90);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Registration_button">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>410</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgba(255, 255, 255, 0);
	color: rgb(182, 182, 182);
}
QPushButton:hover
{		
	color: rgb(220, 220, 220);
}</string>
   </property>
   <property name="text">
    <string>Don't have an account? Registration now</string>
   </property>
  </widget>
  <widget class="QLabel" name="Logo_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ApplicationIcons/Logotype.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Sign_in_button">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>303</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 rgb(0, 200, 250), 
				stop: 0.4 rgb(30, 150, 255),
                stop:1 rgb(60, 90, 255));
	border: 0px;
}
QPushButton:hover
{
	
	background-color: rgb(25, 206, 255);
}</string>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
   <property name="iconSize">
    <size>
     <width>220</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoRepeatDelay">
    <number>0</number>
   </property>
   <property name="autoRepeatInterval">
    <number>-4</number>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Error_window_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>350</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 50, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Remember_me_button">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>260</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>0</number>
   </property>
   <property name="autoRepeatInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_button">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgba(255, 255, 255, 0);
	border: 0px; 
}
QPushButton:hover
{ 
	background-color: rgb(249, 22, 34);
 }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Enter_by_email_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(25, 180, 255);</string>
   </property>
   <property name="text">
    <string>ENTER BY EMAIL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>GAMES STORE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
